frontend/src/store/store.ts

```diff
@@ -0,0 +1,7 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+import { rootReducer } from './root-reducer';
+
+export const store = configureStore({
+  reducer: rootReducer,
+});
```

Comment:
It's a better practice to keep the `combineReducers` (declared in `frontend/src/store/root-reducer.ts` now) in the same file before the `configureStore`. It'll make store files reading easier